packages/core/src/scrim/Scrim.tsx

```diff
@@ -24,13 +18,15 @@ import "./Scrim.css";
 
 const scrims = new Set();
 
-const defaultParent = typeof document !== "undefined" ? document.body : null;
+const defaultParent = () =>
+  typeof document !== "undefined" ? document.body : null;
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
 const noop = () => {};
 
 const withBaseName = makePrefixer("uitkScrim");
 
-function preventSelection(parent = defaultParent): () => void {
+function preventSelection(parent = defaultParent()): () => void {
```

Comment:
I wanted this to be evaluated every time the function runs and not just when the module is loaded, like it was before, in case of server side rendering.

Reply:
👍 makes sense. Maybe should be getDefaultParent then

Reply:
Sounds good